wesnoth-wof.po

```diff
@@ -3153,11 +3236,16 @@ msgid ""
 "I wish to dispatch a sky patrol as well.\n"
 "To begin ceaseless watch of the Greatland just off our eastern shore."
 msgstr ""
+"Jak poroučíš.\n"
+"\n"
+"Chtěl bych také vyslat nebeskou hlídku.\n"
+"Abych začali nepřetržitě sledovat Velkou zemi kousek od našeho východního "
```

Comment:
```suggestion
"Aby začali nepřetržitě sledovat Velkou zemi kousek od našeho východního "
```

Reply:
Hm, ono nejlepší by bylo Aby začala. Aby začali není špatně, ale není to tak „hladké“.